e2e/_screenshot.js

```diff
@@ -0,0 +1,53 @@
+module.exports = screenshot;
+
+var path = require('path');
+var child_process  = require('child_process');
+var mapnik = require('mapnik');
+
+var SHA = process.env.TRAVIS_COMMIT;
+
+function screenshot(id) {
+  var screenshotPath = path.resolve(__dirname, '..', 'screenshots', id + '.screenshot.png');
+  var screenshotUrl = `https://media.githubusercontent.com/media/angular/visualdiff-dev/${ SHA }/screenshots/${ encodeURIComponent(id) }.screenshot.png`;
```

Comment:
Could you shorten this line by breaking the `https://media.githubusercontent.com/media/angular/visualdiff-dev/` into a separate const near the top of the file?